doc/modules/integrating-applications/p_configure-publish-api-provider-quickstart.adoc

```diff
@@ -28,71 +37,85 @@ same as the simple integrations that you already published.
 .Procedure
 
 . Import the *TaskAPI* quickstart integration:
-.. Go to 
+ifndef::getting-started-with-apis-in-red-hat-integration[]
+.. Go to
 link:{syndesis-quickstart-url}/api-provider[]
-and download `TaskAPI-export.zip`. 
-.. In {prodname}, in the left navigation panel, click *Integrations*. 
-.. In the upper right, click *Import*. 
-.. Drag and drop the `TaskAPI-export.zip` file that you downloaded. {prodname} indicates
-that it has successfully imported the file. 
+and download `TaskAPI-export.zip`.
+endif::getting-started-with-apis-in-red-hat-integration[]
+.. In {prodname}, in the left navigation panel, click *Integrations*.
+.. In the upper right, click *Import*.
+.. Drag the `TaskAPI-export.zip` file that you downloaded to the *Import* page. {prodname} indicates
+that it has successfully imported the file.
 .. In the left navigation panel, click *Integrations* to see
 an entry for the *TaskAPI* integration that you just imported. Although the entry
-indicates that configuration is required, this integration is ready to publish. 
+indicates that configuration is required, this integration is ready to publish.
 
-. In the *TaskAPI* entry, click image:images/tutorials/ThreeVerticalDotsKebab.png[Three Vertical Dots] and 
+. In the *TaskAPI* entry, click image:images/integrating-applications/ThreeVerticalDotsKebab.png[Three Vertical Dots] and
 then click *Edit* to display a list
-of the operations that this API provides. 
+of the operations that this API provides.
+
+ifdef::getting-started-with-apis-in-red-hat-integration[]
++
+Each operation has a flow defined for it. In {prodname}, a _flow_ defines the connections and steps to execute for each REST operation in the integration. The imported quickstart provides pre-defined flows for each operation.
```

Comment:
Since a connection is one kind of step, this should be "connections and other steps to execute...."